tensorboard/notebook.py

```diff
@@ -151,6 +151,7 @@ def print_or_update(message):
     else:
       handle.update(IPython.display.Pretty(message))
 
+  args_string='%s %s' % (_get_context_default_args(), args_string)
```

Comment:
I’m slightly hesitant to add this because (a) it breaks the property
that “`%tensorboard ARGS` is the same as running `tensorboard ARGS` at a
shell”, which greatly simplifies the mental model, and (b) it makes it
impossible to reuse existing TensorBoard instances started outside of
the notebook mode (and for a non-obvious reason: see point (a)).
Similarly, `notebook.display(port=1234)` won’t be fixed by this change,
a further inconsistency.

I’ll think about this a bit and see if I can convince myself that it’s
fine. :-)

Reply:
Ah, I totally forgot class of people who would be doing `notebook.display`. I will, too, have to think about this.

When I originally wrote this, I conceived it as context aware defaults but I can see both (a) and (b) tricky. I will sleep on it on more night :)